Documentation/teaching/info/vm.rst

```diff
@@ -44,7 +44,7 @@ tools/labs:
    char device redirected to /dev/pts/19 (label virtiocon0)
```

Comment:
@doringeman change looks good to me. Only, few things to consider.

- Please add `Documentation: ` prefix to the commit subject. E.g `Documentation: Change QEMU_DISPLAY from sdl to gtk`
- Please explain, in the commit message, why this change is needed. 
The explanation given in the issue description https://github.com/linux-kernel-labs/linux/issues/232 should be pasted in the commit message because on the long run people will mostly look at git history.

Reply:
Solved in https://github.com/linux-kernel-labs/linux/pull/233/commits/b382e63aa3d2d0b9ef91a1ee50bf9856de38f4ab.

Reply:
Solved and also revamped 2 other files in https://github.com/linux-kernel-labs/linux/commit/80345f23855695a92f81fec6bd1c00548f4739bd.